### PR TITLE
[HPRO-1182] Added NPH Participant Review Pages and Tests.

### PR DESCRIPTION
**Pull request by Ehaic**
| Q                   | A
| ------------------- | --------------
| Target Release?     | 3.0.0 
| Bug fix?            | ❌ 
| New feature?        | ✅ 
| Database migration? | ❌ 
| New configuration?  | ❌ 
| Composer updates?   | ❌ 
| NPM updates?        | ❌ 
| Jira Ticket(s)      | HPRO-1182

### Screenshots 
image

### CODE REVIEW

**Comment by shyamzcode** [comment]
I think in todays participant page we need to display unique participants. Participant ids and name should be grouped together like in the AOU Today's participant page

image

AOU Today's participant page
image

**Comment by shyamzcode** [comment]
Looks like the search is not working as expected. When I enter today's date and submit I get empty results even though there are participants.
image

File: templates/program/nph/review/today.html.twig [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+{% block body %}
+    {% include 'program/nph/review/page_heading.html.twig' %}
+    <div class="well today-filter-well" data-title="Filter">
+        {{ form_start(todayFilterForm, { attr: { class: 'form-inline', autoComplete: 'off' } }) }}
+        {{ form_errors(todayFilterForm) }}
+        <div class="today-filter" align="center">
+            {{ form_rest(todayFilterForm) }}
+            <div class="form-group">
+                <a href="{{ path('review_today') }}" class="btn btn-default">Reset</a>
```
Comment (shyamzcode) [inline templates/program/nph/review/today.html.twig:15]: This should redirect to nph_review_today route.

**Review by shyamzcode — CHANGES_REQUESTED** [review]
I commented few things that I noticed. Rest looks good.

**Comment by Ehaic** [comment]
> I think in todays participant page we need to display unique participants. Participant ids and name should be grouped together like in the AOU Today's participant page
> 
> image
> 
> AOU Today's participant page image

This is how the mockup show's it, things are grouped by the order not the participant. https://www.figma.com/file/BxLo0kukva3gdj2T0AD60V/NPH?node-id=1588%3A8779

File: migrations/Version20230210142836.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
```
Comment (shyamzcode) [inline migrations/Version20230210142836.php:12]: Could you remove these auto generated comments?

**Review by shyamzcode — CHANGES_REQUESTED** [review]
Left a comment regarding the auto generated comments. Rest looks good!

File: migrations/Version20230210142836.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        $this->addSql('CREATE TABLE nph_field_sort (id int auto_increment not null primary key, field_value varchar(100) not null, sort_order int not null)');
+        $this->addSql("INSERT INTO nph_field_sort (field_value, sort_order)
+                            VALUES ('preLMT', 1), ('minus15min', 2), ('minus5min', 3), ('15min', 4), ('30min', 5), ('60min', 6), ('90min', 7), ('120min', 8),
+                                   ('240min', 9), ('postLMT', 10), ('orangeDiet', 1), ('orangeDLW', 2), ('orangeDSMT', 3), ('orangeLMT', 4), ('blueDiet', 5),
+                                   ('blueDLW', 6), ('blueDSMT', 7), ('blueLMT', 8), ('purpleDiet', 9), ('purpleDLW', 10), ('purpleDSMT', 11), ('purpleLMT', 12);");
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
```
Comment (shyamzcode) [inline migrations/Version20230210142836.php:29]: @Ehaic Could you remove this as well?

**Review by shyamzcode — APPROVED** [review]

**Pull request merged**